conftest.py

```diff
@@ -49,7 +49,7 @@
 @pytest.fixture(
     params=["empty_deployment"]
     if os.environ.get("KCIDB_DEPLOYMENT", "") == "This deployment is empty"
-    else []
+    else [], scope="session"
```

Comment:
Could you put the addition on its own line?

Reply:
done